mmv1/products/storage/ManagedFolder.yaml

```
@@ -0,0 +1,73 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ManagedFolder'
+kind: 'storage#ManagedFolder'
+base_url: b/{{bucket}}/managedFolders
+self_link: b/{{bucket}}/managedFolders/{{name}}
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/storage/docs/managed-folders'
+  api: 'https://cloud.google.com/storage/docs/json_api/v1/managedFolder'
+description: |
+  The ManagedFolder resource represents a managed folder in Cloud Storage.
+  You can apply IAM policies on a managed folder to control access to the resources within the managed folder.
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'google_storage_managed_folder'
+    primary_resource_id: 'managed_folder'
+    primary_resource_name: 'fmt.Sprintf("tf-test-bucket-name%s", context["random_suffix"]), fmt.Sprintf("tf-test-managed-folder-name%s", context["random_suffix"])'
+    vars:
+      bucket_name: 'bucket-name'
+      managed_folder_name: "managed-folder-name"
```

```suggestion
    primary_resource_name: 'fmt.Sprintf("tf-test-bucket-name%s", context["random_suffix"]), fmt.Sprintf("tf-test-managed/folder/name%s", context["random_suffix"])'
    vars:
      bucket_name: 'bucket-name'
      managed_folder_name: "managed/folder/name"
```

Might be good to exercise this for nested folders.

---

solved in last commit